CHANGELOG.md

```diff
@@ -1,5 +1,81 @@
 # osquery Changelog
 
+<a name="5.4.0"></a>
+## [5.4.0](https://github.com/osquery/osquery/releases/tag/5.4.0)
+
+[Git Commits](https://github.com/osquery/osquery/compare/5.3.0...5.4.0)
+
+Representing commits from 15 contributors! Thank you all.
+
```

Comment:
If there were a one-liner that would fit in a tweet by our osquery account, what would it be? I suggest "osquery is moving to monthly releases! Version 5.4.0 adds faster file-event monitoring on macOS, a table for enumerating Windows Updates on your hosts, and fixes an AWS IMDSv2 connectivity regression in osquery 5.3" as these are all the big important things I think users should know.

Reply:
I don't think I want to advertise moving to monthly releases. Let's see if we can manage it first :laugh: